address-space-controller/image_rhel.yaml

```diff
@@ -37,16 +37,16 @@ packages:
         - openssl        
 
 artifacts:
-    - path: address-controller-0.19.0.amqmaas-001002-redhat-3.jar
+    - path: address-space-controller.jar
       md5: e6ae8ded8b6062cbd911a826ed34cf96
```

Comment:
Is the md5 still correct?

Reply:
neither the jar filename and md5 is correct. There is no productized version with the renamed artifact yet, so this will have to be updated once it is built.

Reply:
@vbusch I'm fine with holding off merging until the productized version has been built.